File: archive-application/src/main/java/site/archive/dto/v1/user/UserEmailRequestDtoV1.java
package site.archive.dto.v1.user;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;

import jakarta.validation.constraints.Email;
import jakarta.validation.constraints.NotBlank;
import jakarta.validation.constraints.NotNull;

@Getter
@NoArgsConstructor
@AllArgsConstructor
public class UserEmailRequestDtoV1 {

    @Email
    @NotNull
    @NotBlank
    private String email;

}

File: archive-application/src/main/kotlin/site/archive/dto/v1/user/UserEmailRequestDtoV1.kt
package site.archive.dto.v1.user

import jakarta.validation.constraints.Email
import jakarta.validation.constraints.NotBlank

data class UserEmailRequestDtoV1(
    @field: NotBlank(message = "이메일은 필수 입력 항목입니다.")
    @field: Email(message = "올바른 이메일을 입력해 주세요.")
    val email: String
)